### ISSUE TITLE
webUI autocomplete in 2.0.2

### ISSUE THREAD

**Issue by thaiat** [rethinkdb/rethinkdb#4304]
The autocomplete feature does not work anymore after upgrading to 2.0.2

**Comment by thaiat** [comment]
there is an error in the chrome debugger:
```
Uncaught TypeError: Cannot read property 'length' of undefinedContainer.Container.set_doc_description @ cluster-min.js?v=2.0.2:5156Container.Container.set_docs @ cluster-min.js?v=2.0.2:5208(anonymous function) @ cluster-min.js?v=2.0.2:12039f.Callbacks.o @ jquery-1.7.2.min.js?v=2.0.2:2f.Callbacks.p.fireWith @ jquery-1.7.2.min.js?v=2.0.2:2e.extend.ready @ jquery-1.7.2.min.js?v=2.0.2:2c.addEventListener.B @ jquery-1.7.2.min.js?v=2.0.2:2
```

**Comment by latesh** [comment]
Duplicate of #4261 
Close this and follow the above mentioned issue

**Comment by deontologician** [comment]
As latesh mentioned. It's a duplicate, but thanks for reporting it @thaiat ! The fix for this has been approved, and should be released relatively soon (2.0.3)
